.github/workflows/workflow.yaml

```diff
@@ -65,29 +65,35 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gatekeeper: [ "release-3.13", "release-3.14" ]
-    name: "Integration test on Gatekeeper ${{ matrix.gatekeeper }}"
+        gatekeeper: [ "3.15.1", "3.16.0" ]
+        engine: [ "cel", "rego" ]
+    name: "Integration test on Gatekeeper ${{ matrix.gatekeeper }} for ${{ matrix.engine }} policies"
     steps:
       - name: Harden Runner
+        if: ${{ (matrix.gatekeeper == '3.15.1' && matrix.engine == 'rego') || matrix.gatekeeper == '3.16.0' }} # remove this condition once 3.17 is out
```

Comment:
why do we need this for harden runner?

Reply:
nm this is because of the matrix

Reply:
can we do this opposite? if gk 3.15 and cel, we exit 0

Reply:
@sozercan seems like there is no way to do exit for the whole job from a step - https://github.com/orgs/community/discussions/82744. We would need if condition for all steps to skip it regardless.